stable/postgresql/templates/svc.yaml

```diff
@@ -14,6 +14,11 @@ metadata:
 {{- end }}
 spec:
   type: {{ .Values.service.type }}
+  {{- if eq .Values.service.type "ClusterIP" }}
+  {{- if .Values.service.clusterIP }}
```

Comment:
Is it possible to use just a single if instead?

Eg:
```
{{- if and (eq .Values.service.type "ClusterIP") .Values.service.clusterIP }}
```

Reply:
will do,